system-tests/ft-suite/src/test/kotlin/com/hotels/styx/routing/RoutingRestApiSpec.kt

```diff
@@ -120,20 +120,18 @@ class RoutingRestApiSpec : StringSpec() {
                         it!!.status() shouldBe OK
                         it.bodyAs(UTF_8).trim() shouldBe """
                                 ---
-                                name: "responder"
-                                type: "StaticResponseHandler"
-                                tags: []
-                                config:
-                                  status: 200
-                                  content: "Responder"
-
-                                ---
-                                name: "root"
-                                type: "StaticResponseHandler"
-                                tags: []
-                                config:
-                                  status: 200
-                                  content: "Root"
+                                - name: "responder"
+                                  type: "StaticResponseHandler"
+                                  tags: []
+                                  config:
+                                    status: 200
+                                    content: "Responder"
+                                - name: "root"
+                                  type: "StaticResponseHandler"
+                                  tags: []
+                                  config:
+                                    status: 200
+                                    content: "Root"
```

Comment:
Hmm... 🤔 

So conceptually this is a map. Also in styx yaml configuration this is represented as a map. But now this implementation renders it as a list.

For consistency, I would prefer the admin interface also to render it as a map.

Reply:
Yes, of course. I should have noticed that the config yaml uses maps, not lists here.

Reply:
For the case where we are requesting a single object, should it be as now (with `name` as an attribute) or in a map format (with `name` as the key)?

Reply:
Eh, forget that, it should be the single object payload with no `name:` attribute and no key.